src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    public static String getSalaryInfo(String[] names, String[] data,
+                                       String dateFrom, String dateTo) {
+        String dateFromComparable = convertDateToComparable(dateFrom);
```

Comment:
If you create a formatter, make it a constant field. This will help with maintainability and performance if the formatter is used multiple times.